metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/utils/ParserUtils.java

```diff
@@ -55,7 +55,7 @@ public static Long convertToEpoch(String m, String d, String ts,
     Date date = new SimpleDateFormat("MMM", Locale.ENGLISH).parse(m);
     Calendar cal = Calendar.getInstance();
     cal.setTime(date);
-    String month = String.valueOf(cal.get(Calendar.MONTH));
+    String month = String.valueOf(cal.get(Calendar.MONTH) + 1);
```

Comment:
While you're in here, could you add a javadoc comment to the method explaining what the expected inputs look like?  It's not obvious from looking at the function what the expectations are.